Program.cs

```diff
@@ -1,63 +1,86 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
-using Microsoft.Azure.Management.Fluent;
-using Microsoft.Azure.Management.ResourceManager.Fluent;
-using Microsoft.Azure.Management.ResourceManager.Fluent.Core;
-using Microsoft.Azure.Management.ResourceManager.Fluent.Models;
+using Azure;
+using Azure.Core;
+using Azure.Identity;
+using Azure.ResourceManager;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.Resources.Models;
 using Microsoft.Azure.Management.Samples.Common;

```

Comment:
If all the package we referenced is T2, this Sample utility namespace should rename to T2 style

Reply:
The `Common` namespace now is only show the logs on console. Instead of changing the name of it, I decide to remove it and use `Console.WriteLine` directly to write information which makes this sample more simple and easy to understand.